src/domain/eventSearch/constants.ts

```diff
@@ -152,27 +331,38 @@ export const MAPPED_EVENT_CATEGORIES: Record<string, string> = {
 
 //todo: replace with valid keyword ids
 export const MAPPED_COURSE_CATEGORIES: Record<string, string> = {
-  [EVENT_CATEGORIES.MOVIE]: 'yso:p1235',
+  [COURSE_CATEGORIES.MOVIE]: MOVIES_AND_MEDIA_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.LANGUAGES]: LANGUAGES_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.LITERATURE]: LITERATURE_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.ARTS_AND_CULTURE]: ARTS_AND_CULTURE_COURSES_KEYWORDS.join(
+    ','
+  ),
+  [COURSE_CATEGORIES.VISUAL_ARTS]: VISUAL_ARTS_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.HANDICRAFTS]: HANDICRAFTS_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.SPORT]: SPORT_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.MUSIC]: MUSIC_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.GAMES]: GAMES_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.FOOD]: FOOD_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.DANCE]: DANCE_COURSES_KEYWORDS.join(','),
+  [COURSE_CATEGORIES.THEATRE]: THEATRE_COURSES_KEYWORDS.join(','),
```

Comment:
Do we need to join these into a string? Only thing we are doing eith these is splitting them back into array if I see correctly?

Reply:
actually i dont know. when i tried the old logic, then it was not splitting correctly categories. it was creating array like ["kulke1", "kulke2,kulke3"]. when i noticed that i applied this approach to make it ["kulke1, "kulke2", kulke3"]. it was the problem in event categories logic

Reply:
ok strange